velox/common/memory/ByteStream.h

```diff
@@ -223,6 +223,8 @@ class ByteStream {
     append(folly::Range(&value, 1));
   }
 
+  size_t flushSize();
```

Comment:
Would you document this method? 

Make it const?

Reply:
It would be nice to add a test for this new method.

Reply:
No, because updateEnd will update the field `lastRangeEnd_`

Reply:
This methods seems to me the same as ByteStream::size() (except it updates lastRangeEnd_, but it is not clear why it needs to do that). Any particular reason size() cannot be used?

Reply:
In this test, https://github.com/facebookincubator/velox/pull/5493/files#diff-74218bb1911f1f7cb1afd6a27dc1be00fa7f8446f33e16390b2f7c8ee84a2ea5R169

If isBits= true, the size() and flushSize() result are different.